Floofbot/Modules/TagCommands.cs

```diff
@@ -77,6 +77,34 @@ public async Task RequireAdmin([Summary("True/False")] string requireAdmin = "")
             }
         }
 
+        [Command("updatetagnames")]
+        [Summary("Update tag names to new schema")]
+        [RequireUserPermission(GuildPermission.Administrator)]
+        public async Task UpdateTagNames(string dummyParam="")
```

Comment:
@ Reviewers: Any idea what to do here instead of dummyParam?
If I take off this parameter, even if I set the command's priority to 0, the general `.tag` command gets called and says it can't find a tag with name `updatetagnames`.

Reply:
If this is only a one time command, why not just not put it under the tag group? make it a command of its own